scintilla/lexilla/src/Lexilla.cxx

```diff
@@ -178,6 +178,7 @@ static void AddGeanyLexers()
 		&lmBash,
 		&lmBatch,
 		&lmCaml,
+		&lmCIL,
```

Comment:
`AddGeanyLexers()` is not part of Lexilla, it is added when Lexilla is updated by `scintilla_changes.patch` so you need to add there as well, or you lexer will be unknown and the crash will return on the next Lexilla update.

Reply:
Thanks!

Fixed: 1397da184180e39a2797ffeb9f15c5174f1f52c5